nifi-registry-core/nifi-registry-data-model/src/main/java/org/apache/nifi/registry/authorization/CurrentUser.java

```diff
@@ -44,6 +45,15 @@ public void setAnonymous(boolean anonymous) {
         this.anonymous = anonymous;
     }
 
+    @ApiModelProperty(value = "Indicates if the NiFi instance supports logging in")
+    public boolean getLoginSupported() {
```

Comment:
Minor quibble, but would prefer the method name `isLoginSupported` to be consistent with `isAnonymous`.

Reply:
Makes sense, will update